stubs/docutils/docutils/frontend.pyi

```diff
@@ -60,6 +61,14 @@ class OptionParser(optparse.OptionParser, SettingsSpec):
     default_error_encoding_error_handler: ClassVar[str]
     config_section: ClassVar[str]
     version_template: ClassVar[str]
+    def __init__(
+        self,
+        components: Tuple[Type[Parser]] = ...,
+        defaults: Dict[str, object] = ...,
+        read_config_files: bool = ...,
```

Comment:
As this has an explicit default of `None`:

```suggestion
        read_config_files: bool | None = ...,
```

Reply:
Good catch! My bad.